common/dev/isl69260.c

```diff
@@ -0,0 +1,76 @@
+#include "isl69260.h"
+
+#include <stdio.h>
+
+#include "sensor.h"
+#include "hal_i2c.h"
+#include "plat_sensor_table.h"
+
+#define RETRY 5
+
+struct isl69260_page_data isl69260_page_data_args[] = { [0] = { 0x0, 0x0 }, [1] = { 0x0, 0x1 } };
+
+int isl69260_switch_page(uint8_t sensor_num, void *args)
+{
+	snr_cfg *cfg = &sensor_config[sensor_config_index_map[sensor_num]];
+	struct isl69260_page_data *page = args;
+	I2C_MSG msg;
+	msg.bus = cfg->port;
+	msg.target_addr = cfg->target_addr;
+	msg.data[0] = page->data0;
+	msg.data[1] = page->data1;
+	msg.tx_len = 2;
+
+	if (i2c_master_write(&msg, RETRY)) {
+		printf("sensor num %x switch page fail\n", sensor_num);
+		return -1;
+	}
+
+	return 0;
+}
+
+bool isl69260_read(uint8_t sensor_num, float *reading)
```

Comment:
check readings != NULL

Reply:
The optimization code is based on v2022.07.01. After that, the main branch has been phased in some
commits related to sensor monitoring. Those changes will be phased in main_v2 in the other PRs. This
problem will be solved in those PRs.
Please check #223  .